### ISSUE TITLE
Akahu SDK uses Axios - not compatible with Cloudflare Workers

### ISSUE THREAD

**Issue by HexaCubist** [akahu-io/akahu-sdk-js#40]
Upon trying to deploy an app using Akahu to Cloudflare, I got the following error: 

```
        "TypeError: adapter is not a function"
```

Following some Google search results, it appears that this is because Axios uses Node-native APIs that are not available on Cloudflare Workers (https://community.cloudflare.com/t/can-i-use-axios-in-a-worker/168139). What is the potential of providing support for Fetch?

**Comment by dabrowne** [comment]
Hi @HexaCubist I've just published `akahu@1.14` which exposes the Axios `adapter` option. This should allow you to use the solution described here: https://hoyle.io/databases/writing-database/using-orysh-with-cloudflare-workers


```sh
npm install @haverstack/axios-fetch-adapter
```

```typescript
import { AkahuClient } from "akahu";
import fetchAdapter from "@haverstack/axios-fetch-adapter";

const akahu = new AkahuClient({
  appToken: "...",
  appSecret: "...",
  adapter: fetchAdapter,
});
```

**Comment by HexaCubist** [comment]
Great! Thank you, that works a treat. Because I'm using Vite, I had to install `vite-plugin-cjs-interop` to use this adapter, but once I had that sorted, everything clicked together perfectly, and I'm now running on edge functions. Thanks for your very quick response!
